pkg/machines/components/vmSnapshotsTab.jsx

```diff
@@ -146,6 +153,49 @@ class VmSnapshotsTab extends React.Component {
                     );
                 }
             },
+            {
+                name: "", value: (snap, snapId) => {
+                    const revertSnapshotHelper = () => {
+                        const revertDialogProps = {
+                            idPrefix: `${id}-snapshot-${snapId}-revert`,
+                            vm,
+                            snap,
+                            onClose: () => this.setState({ revertDialogProps: undefined }),
+                        };
+                        return (
+                            <Button id={`${id}-snapshot-${snapId}-revert`}
+                                variant='secondary'
+                                onClick={() => this.setState({ revertDialogProps })}>
+                                {_("Revert")}
+                            </Button>
+                        );
+                    };
+
+                    const deleteSnapshotHelper = () => {
+                        const deleteDialogProps = {
+                            objectType: "Snapshot",
+                            objectName: snap.name,
+                            onClose: () => this.setState({ deleteDialogProps: undefined }),
+                            deleteHandler: () => {
+                                return deleteSnapshot({ connectionName: vm.connectionName, domainPath: vm.id, snapshotName: snap.name })
+                                        .then(() => dispatch(getVmSnapshots({ connectionName: vm.connectionName, domainPath: vm.id })));
+                            },
+                        };
+
+                        return (
+                            <DeleteResourceButton objectId={`${id}-snapshot-${snapId}`}
+                                showDialog={() => this.setState({ deleteDialogProps })} />
+                        );
+                    };
+
+                    return (
+                        <div className='machines-listing-actions pull-right'>
```

Comment:
This seems to be why it breaks!
Remove `pull-right`, and you should be good.

Reply:
This is how it looks without that class applied.
![Screenshot from 2020-09-01 16-54-02](https://user-images.githubusercontent.com/209895/91867297-c6dd2f80-ec73-11ea-907d-22d9c4ec44a6.png)

Reply:
Thanks a lot for your help. I fixed it and this is how it looks now (basically the same as in your screenshot): 
![Screenshot from 2020-09-02 02-01-05](https://user-images.githubusercontent.com/42733240/91917486-590b2500-ecc0-11ea-81c6-986a5d13bcc6.png)